src/nspawn/nspawn-seccomp.c

```diff
@@ -153,7 +152,7 @@ static int seccomp_add_default_syscall_filter(
         }
 
         STRV_FOREACH(p, syscall_whitelist) {
-                r = seccomp_add_syscall_filter_item(ctx, *p, SCMP_ACT_ALLOW, syscall_blacklist, false);
+                r = seccomp_add_syscall_filter_item(ctx, *p, SCMP_ACT_ALLOW, syscall_blacklist, true);
```

Comment:
(hmm, what was the rationale for making the logging optional in the first place? I mean debug logging should always be OK?)

Reply:
True, I shouldn't have done that.